apps/applets/hello/base/deployment.yaml

```
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hello
+  namespace: applets-hello
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: hello
+  template:
+    metadata:
+      labels:
+        app: hello
+      annotations:
+        vault.hashicorp.com/agent-inject: "true"
+        vault.hashicorp.com/tls-skip-verify: "true"
+        vault.hashicorp.com/role: "secret-reader"
+        # https://github.com/ApplETS/Backend-Hello/blob/main/core/.env.template
+        vault.hashicorp.com/agent-inject-template-hello-env: |
+          {{- with secret "kv/applets/hello-env" -}}
+            export CONNECTION_STRING="{{ .Data.data.connection_string }}"
+            export REDIS_CONNECTION_STRING="{{ .Data.data.redis_connection_string }}"
+            export ASPNETCORE_ENVIRONMENT="{{ .Data.data.aspnetcore_environment }}"
+            export FRONTEND_BASE_URL="{{ .Data.data.frontend_base_url }}"
+            export CDN_URL="{{ .Data.data.cdn_url }}"
+          {{- end }}
+    spec:
+      serviceAccountName: secrets-reader
+      containers:
+        - name: hello
+          image: ghcr.io/applets/backend-hello:main
+          imagePullPolicy: Always
+          command: ['/bin/sh', '-c', 'source /vault/secrets/hello-env && dotnet api.core.dll']
+          ports:
+            - containerPort: 8080
+          volumeMounts:
```

Adapter selon la prevision de consommation de ressources et de droits nécessaires

```suggestion
          resources:
            requests:
              cpu: 100m
              memory: 128Mi
            limits:
              cpu: 500m
              memory: 256Mi
              ephemeral-storage: 1Gi
          securityContext:
            allowPrivilegeEscalation: false
            readOnlyRootFilesystem: false
            runAsNonRoot: true
            runAsUser: 10001
            runAsGroup: 10001
            capabilities:
              drop:
                - ALL
            seccompProfile:
              type: RuntimeDefault
          volumeMounts:
```